cli/src/commands/prev.rs

```diff
@@ -78,25 +77,12 @@ pub(crate) fn cmd_prev(
             .heads()
             .contains(current_wc_id);
     let current_wc = workspace_command.repo().store().get_commit(current_wc_id)?;
-    let start_id = if edit {
-        current_wc_id
-    } else {
-        match current_wc.parent_ids() {
-            [parent_id] => parent_id,
-            _ => return Err(user_error("Cannot run `jj prev` on a merge commit")),
-        }
-    };
-    let ancestor_expression = RevsetExpression::commit(start_id.clone()).ancestors_at(offset);
-    let target_revset = if edit {
-        ancestor_expression
-    } else {
-        // Jujutsu will always create a new commit for prev, even where Mercurial cannot
-        // and fails. The decision and all discussion around it are available
-        // here: https://github.com/martinvonz/jj/pull/1200#discussion_r1298623933
-        //
-        // If users ever request erroring out, add `.ancestors()` to the revset below.
-        ancestor_expression.minus(&RevsetExpression::commit(current_wc_id.clone()))
```

Comment:
I'm not sure if we still need this comment or not, or where would be a better place to put it. Maybe it belongs above the call to `tx.check_out` further down?

Reply:
Ok, after looking at https://github.com/martinvonz/jj/pull/1200#discussion_r1253946006, where the check originally comes from, it'd be better to prompt here instead of just skipping the commit and dropping the optional check.

Reply:
cc @martinvonz, do you remember the intention of it?

Reply:
I guess I don't really understand the comment in the first place. In what scenario might we want to return an error?

Reply:
No, I don't know what that comment was about either.

Reply:
I wasn't interested in the comment, but why we'd use the parent ids in the revset. It was your suggestion back then.